gluon/utf8.py

```diff
@@ -10,12 +10,15 @@
 Utilities and class for UTF8 strings managing
 ----------------------------------------------
 """
-import __builtin__
+from __future__ import print_function
+from gluon._compat import builtin as __builtin__, unicodeT, iteritems
+
 __all__ = ['Utf8']
 
 repr_escape_tab = {}
-for i in range(1, 32):
-    repr_escape_tab[i] = ur'\x%02x' % i
+#FIXME PY3
+#for i in range(1, 32):
```

Comment:
It raises a syntax error on py3 and I commented it out while working. It should be re-enabled before releasing a new stable for py2.7.